instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -9,36 +9,32 @@ module Instrumentation
     module AwsSdk
       # Generates Spans for all interactions with AwsSdk
       class Handler < Seahorse::Client::Handler
-        SQS_SEND_MESSAGE = 'SQS.SendMessage'
-        SQS_SEND_MESSAGE_BATCH = 'SQS.SendMessageBatch'
-        SQS_RECEIVE_MESSAGE = 'SQS.ReceiveMessage'
-        SNS_PUBLISH = 'SNS.Publish'
-
         def call(context)
           return super unless context
 
-          service_name = service_name(context)
+          service_id = service_name(context)
           operation = context.operation&.name
-          client_method = "#{service_name}.#{operation}"
-          attributes = {
-            'aws.region' => context.config.region,
-            OpenTelemetry::SemanticConventions::Trace::RPC_SYSTEM => 'aws-api',
-            OpenTelemetry::SemanticConventions::Trace::RPC_METHOD => operation,
-            OpenTelemetry::SemanticConventions::Trace::RPC_SERVICE => service_name
-          }
-          attributes[SemanticConventions::Trace::DB_SYSTEM] = 'dynamodb' if service_name == 'DynamoDB'
-          MessagingHelper.apply_sqs_attributes(attributes, context, client_method) if service_name == 'SQS'
-          MessagingHelper.apply_sns_attributes(attributes, context, client_method) if service_name == 'SNS'
+          client_method = "#{service_id}.#{operation}"
```

Comment:
In theory, I think all service clients should have an operation (and its name) on the `context` object (at least for V3). I'm not too sure about V2 since that has been long deprecated but I imagined it should be same since we use service models to generate the clients.

However, I will lean to leave this as it is - since I want to keep the logic the same much as it can to keep the legacy behavior before i introduce the integration with our recent support of Observability.